oshi-core/src/main/java/oshi/software/os/windows/WindowsOperatingSystem.java

```diff
@@ -770,4 +770,34 @@ private static void enableDebugPrivilege() {
         }
         Kernel32.INSTANCE.CloseHandle(hToken.getValue());
     }
+
+    /** {@inheritDoc} */
+    @Override
+    public OSService[] getServices() {
+        W32ServiceManager sm = new W32ServiceManager();
+        sm.open(Winsvc.SC_MANAGER_ALL_ACCESS);
```

Comment:
This fails without running as administrator.  Use `SC_MANAGER_ENUMERATE_SERVICE `.  I think we might want a try with resources block anyway, to catch `com.sun.jna.platform.win32.Win32Exception` here.